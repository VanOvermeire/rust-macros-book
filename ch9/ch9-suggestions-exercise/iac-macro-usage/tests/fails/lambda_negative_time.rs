use iac_macro::iac;

fn main() {
    iac! {
        lambda name mem -10
    }
}
