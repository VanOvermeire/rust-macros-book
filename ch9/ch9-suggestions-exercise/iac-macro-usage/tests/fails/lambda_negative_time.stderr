error: memory needs a positive value <= 10240
 --> tests/fails/lambda_negative_time.rs:5:25
  |
5 |         lambda name mem -10
  |                         ^
